docs/index.md

```diff
@@ -14,6 +14,7 @@ Exporting and importing a sub-tree, or a single page is supported.
 
 - Related objects (such as images, documents, and snippets) are not translated or duplicated.
 - Pages will be in draft mode. Pages need to be reviewed and published.
+- Nested streamfield are known to cause issues since their database values are not correct. The page will be created but causes errors.
```

Comment:
We call them nested streamfields, but they are `StreamBlock`s right?

And do they always fail? Or after a certain level of nesting?

Where do these errors occur? `causes errors` -> `cause errors on upload` or something.

Reply:
StreamBlocks indeed, thanks for pointing out ;)